sdk/highlight-next/package.json

```diff
@@ -7,14 +7,20 @@
 	"types": "./dist/index.d.ts",
 	"exports": {
 		".": {
+			"edge": "./dist/index.edge.js",
+			"edge-light": "./dist/index.edge.js",
+			"browser": "./dist/index.edge.js",
+			"worker": "./dist/index.edge.js",
+			"workerd": "./dist/index.edge.js",
 			"require": "./dist/index.js",
 			"import": "./dist/index.mjs",
 			"types": "./dist/index.d.ts"
 		}
 	},
 	"scripts": {
-		"typegen": "tsup src/index.ts --dts-only",
-		"build": "tsup src/index.ts --format cjs,esm --dts",
+		"typegen": "tsup src/index.ts --dts-only && tsup src/index.edge.ts --dts-only",
+		"dev": "yarn build --watch",
+		"build": "tsup src/index.edge.ts --format cjs,esm --dts && tsup src/index.ts --format cjs,esm --dts",
```

Comment:
did you mean to commit these changes? if so, we should bump the `version` references here and in all other affected sdk packages accordingly (and update changelog references). would you mind also updating the pr description about how this was tested with and without the app directory?